examples/202005-typescript-project-structure/boat/BUILD

```diff
@@ -0,0 +1,14 @@
+load("@rules_tsc_compile//defs.bzl", "tsc_module")
+load("@rules_mocha//defs.bzl", "mocha_test")
+
+tsc_module(
+    name = "tsconfig",
+    tsconfig = "tsconfig.json",
+    srcs = glob(["src/*.ts"]),
+    tests = glob(["tests/*.ts"])
+)
+
+mocha_test(
+    name = "unit",
+    deps = ["@tsc//src/boat:ts_test_lib"]
+)
```

Comment:
I'm not quite sure how we get here (or rather, I'm not sure what the proper things are to build at the moment) but I think this is how BUILD files ought to end up (roughly) looking.